### ISSUE TITLE
[Adaptive Card iOS - InputLabelPosition.json]: Screen Reader is not conveying the state as Selected for the menu items under the Select color combo box.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards-Mobile#173]
### Target Platforms

iOS

### SDK Version

Version 1.0 (2.4.0-beta.24.5.30.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 17.5.1
Application: Adaptive Cards
App Version: Version 1.0 (2.4.0-beta.24.5.30.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Navigate to "V1.6" control and select it.
4. Navigate to 'Elements' control and activate it.
5. Navigate to 'InputLabelPosition.json' under 'Elements' list item and activate it.
6. Card will be appear.
7. Perform right swipe, navigate to the 'Select Color' combo box and double tap to activate it.
8. Navigate the list and observe the issue.

**Actual Result:**
While navigating the menu items present under 'Select Color' combo box, screen reader is not announcing the state information as selected for the selected menu item. It is announcing as 'Red color 1 of 3.'

**Expected Result:**
Screen Reader should announce the state as Selected for the menu items under the select color combo box. It should announce as 'Red color 1 of 3 selected.'

**User Impact:**
It will impact the screen reader user if the screen reader will not announce the state as selected for the selected menu items.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section.

### Screenshots

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/644a56f7-e31d-4e16-b756-3dc01043fb43

### Card JSON

```JSON
NA
```
